### ISSUE TITLE
replacement for "&rfc.number;" mechanism in xml2rfc

### ISSUE THREAD

**Issue by reschke** [rfc-format/draft-iab-xml2rfc-v3-bis#29]
It just occurred to me that there might be a relatively elegant way to provide this functionality in the vocabulary:

- allow \@anchor on the root element
- special-case \<xref> for references to the root element to insert either the RFC number of the draft name.

**Comment by levkowetz** [comment]
On 2018-02-14 17:11, Julian Reschke wrote:
> It just occurred to me that there might be a relatively elegant way to provide this functionality in the vocabulary:
> 
> - allow \@anchor on the root element
> - special-case \<xref> for references to the root element to insert either the RFC number of the draft name.

Yes, that would work nicely.

	Henrik

**Comment by levkowetz** [comment]
I just tried this out, and there seems to some friction:  An xsd:ID attribute on the document root does not seem to be recognized as a valid ID, so any xsd:IDREF attribute referring to it results in a validation error: IDREF attribute target references an unknown ID "foo"

**Comment by reschke** [comment]
Well, did you add the anchor attribute to the \<rfc> element in the grammar?

**Comment by levkowetz** [comment]
Yes, certainly.  Otherwise I'd have gotten an error for that, too.

**Comment by stpeter** [comment]
The RPC is now accustomed to not having this feature and doesn't see a strong need to add it back in at this time, so I'm marking this as a feature request for future consideration.
